source/css/sass/patterns/templates/listing-grid.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../functions/grid";
+@import "../../mixins/grid";
+@import "../../mixins/media-query";
+@import "../../variables/grid";
+
+.listing-grid {
+  --primary-column-width: #{get-overall-width($grid-main_column_count)};
+
+  @include base-grid($grid-max_width);
+  grid-template-areas: ". primary .";
+  grid-template-columns: [full-start] 1fr [main-start] var(--primary-column-width) [main-end] 1fr [full-end];
+  grid-column-gap: var(--GRID-COLUMN-GAP);
+
+  @include mq($from: x-wide) {
+    $primary-column-count: floor($grid-main_column_count * 0.7);
```

Comment:
Yeah. Whether we do make in a variable... 🤷‍♂️

Reply:
Might be kind to call it out as a magic number in a code comment so that someone doesn't try to track it down (been there before)? I don't feel strongly about it.